index.d.ts

```diff
@@ -55,6 +55,24 @@ type Delay = {
 		}
 	): delay.ClearablePromise<T>;
 
+	/**
+	Create a promise which resolves after a random amount of milliseconds between `minimum` and `maximum` has passed.
+
```

Comment:
Can you elaborate in the docs on the use-case for this method? (Testing)

Reply:
Fixed!